media/css/firefox/whatsnew/whatsnew-70.scss

```diff
@@ -0,0 +1,212 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../protocol/components/state-fxa';
+@import '../../../protocol/css/components/emphasis-box';
+@import '../../../protocol/css/components/notification-bar';
+
+.content-main {
+    padding: 0;
+    @media #{$mq-md} {
+        padding-top: $spacing-2xl;
+    }
+}
+
+.c-section-title-secondary {
+    @include text-display-xs;
+    margin: 0 auto $layout-md;
+    max-width: $content-md - ($layout-md * 2);
+}
+
+.c-utilities {
+    @include text-body-sm;
+    max-width: $content-md;
+    text-align: center;
+}
+
+.mzp-c-cta-link:link {
+    border-bottom: 1px solid;
+    font-weight: normal;
+    text-decoration: none;
+
+    &:hover,
+    &:active,
+    &:focus {
+        @include transition(border-bottom-color 100ms ease-in-out);
+        border-bottom-color: transparent;
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Up-to-date page header
+
+.c-page-header {
+    background: transparent;
+}
+
+.mzp-c-notification-bar {
+    color: $color-off-black;
+    display: none;
+    margin: $layout-xs;
+
+    .is-up-to-date {
+        display: block;
+    }
+}
+
+.c-page-header-inner {
+    align-items: center;
+    display: grid;
+    grid-template-columns: 1fr;
+}
+
+.c-page-header-up-to-date {
+    margin: 0 $layout-sm;
+}
+
+.c-page-header-logo-fx {
+    min-width: 216px;
+}
+
+@media #{$mq-md} {
+    .c-page-header-inner {
+        align-items: center;
+        display: grid;
+        grid-template-columns: 1fr max-content 1fr;
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Emphasis box
+
+.mzp-c-box-emphasis {
+    @include border-box;
+    color: $color-off-black;
+    margin: 0 auto $layout-sm;
+    max-width: $content-md - ($layout-md * 2);
+    padding: $spacing-xl;
+    text-align: center;
+
+    :link {
+        color: $color-off-black;
+    }
+
+    .c-emphasis-box-logo {
+        display: block;
+        margin: 0 auto $spacing-lg;
+        width: 50px;
+    }
+
+    .c-emphasis-box-title {
+        @include text-display-sm;
+
+        span {
+            display: inline;
```

Comment:
That "It’s included with your Firefox account." bit is showing for all users because of this rule. You might need more specificity on a hiding rule, or maybe this rule isn't needed?

Reply:
I included the `inline` property because `show-fxa-supported-signed-in` adds a class of `display: block` which separates the span on a different line. 
writing:
```
body.state-fxa-supported-signed-in {
    span.show-fxa-supported-signed-in {
        display: inline;
    }
}
```
for more specificity.